nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -507,6 +507,8 @@
 
 - `services.rippleDataApi` has been removed, as `ripple-data-api` was broken and had not been updated since 2022.
 
+- `luakit` has been updated to 2.4.0. If you use any website which uses IndexedDB or local storage and wish to retain the saved information, [some manual intervention may be required](https://luakit.github.io/news/luakit-2.4.0.html)
```

Comment:
This is not very actionable. Probably almost no one knows if they use those techs or not. We should rather mention the directories which should be moved.